test/server.test.js

```diff
@@ -0,0 +1,97 @@
+const mysql = require('mysql');
+const db = require('../server/database/index.js');
+const seed = require('../server/database/seed.js');
+
+describe('Database seeding', () => {
+
+  test('Database properly seeds', done => {
+
+    var callback = (listingData, photoData) => {
+      expect(listingData.length).toBe(100);
+      expect(photoData.length).toBeGreaterThan(499);
+      expect(photoData.length).toBeLessThan(1001);
+      done();
+    };
+
+    var dropDB = 'DROP DATABASE IF EXISTS photo_carousel;';
+    var createDB = 'CREATE DATABASE photo_carousel;';
+    var useDB = 'USE photo_carousel;';
+    var createListings = 'CREATE TABLE listings (id INT NOT NULL AUTO_INCREMENT,listing_name VARCHAR(255),PRIMARY KEY (id));';
+    var createPhotos = 'CREATE TABLE photos (id INT NOT NULL AUTO_INCREMENT,photo_url VARCHAR(2083),photo_description VARCHAR(255),listing_id INT,PRIMARY KEY (id),FOREIGN KEY (listing_id) REFERENCES listings(id));';
+
+    var resetDB = (cb) => {
+      db.connection.query(dropDB, () => {
+        db.connection.query(createDB, () => {
+          db.connection.query(useDB, () => {
+            db.connection.query(createListings, () => {
```

Comment:
Maybe rethink through this test so it doesn't drop and create a new db every time.

Reply:
Just kidding, leave it be if it is working for you